lib/internal/modules/cjs/loader.js

```diff
@@ -387,6 +385,36 @@ function resolveExports(nmPath, request, absoluteRequest) {
   return path.resolve(nmPath, request);
 }
 
+function resolveExportsTarget(pkgPath, target, subpath, basePath, mappingKey) {
+  if (typeof target === 'string') {
+    if (target.substr(0, 2) === './' &&
```

Comment:
why the annex b substr instead of slice, or perhaps:
```suggestion
    if (target.startsWith('./') &&
```